src/pages/Layout/index.tsx

```diff
@@ -2,11 +2,32 @@ import { Outlet } from 'react-router-dom';
 import Header from '@/components/Header/Header';
 import Footer from '@/components/footer/footer';
 import style from './Layout.module.scss';
+import { useState } from 'react';
+import BurgerMenu from '@/components/BurgerMenu/BurgerMenu.tsx';
+import { MenuContext } from '@/context/menuContext.ts';
 
 const Layout = () => {
+  const [isMenuOpen, setIsMenuOpen] = useState<boolean>(false);

```

Comment:
This is extra typing; TypeScript types primitive values on its own. You can verify this by removing the code.
`const [isMenuOpen, setIsMenuOpen] = useState(false);`